github.sgnl.yaml

```
@@ -0,0 +1,700 @@
+    pageSize: 100
+    pagesOrderedById: false 
+    attributes:
+      - name: collaboratorId
+        externalId: $.node.id
+        type: String
+        indexed: true
+        uniqueId: true
+        attributeAlias: repositoryCollaboratorId
+      - name: permission
+        externalId: permission
+        type: String
+  # Collaborator is a User entity type: https://docs.github.com/en/graphql/reference/objects#repositorycollaboratoredge
+  Collaborator:
+    displayName: Collaborator
+    externalId: Collaborator
+    description: Entity representing a GitHub User assigned to some Repository.
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: id
+        externalId: id
+        type: String
+        indexed: true
+        uniqueId: true
+        attributeAlias: collaboratorId
+      - name: bio
+        externalId: bio
+        type: String
+      - name: company
+        externalId: company
+        type: String
+      - name: createdAt
+        externalId: createdAt
+        type: DateTime
+      - name: databaseId
+        externalId: databaseId
+        type: Int64
+      - name: email
+        externalId: email
+        type: String
+      - name: login
+        externalId: login
+        type: String
+      - name: isBountyHunter
+        externalId: isBountyHunter
+        type: Bool
+      - name: isCampusExpert
+        externalId: isCampusExpert
+        type: Bool
+      - name: isDeveloperProgramMember
+        externalId: isDeveloperProgramMember
+        type: Bool
+      - name: isEmployee
+        externalId: isEmployee
+        type: Bool
+      - name: isFollowingViewer
+        externalId: isFollowingViewer
+        type: Bool
+      - name: isHireable
+        externalId: isHireable
+        type: Bool
+      - name: isSiteAdmin
+        externalId: isSiteAdmin
+        type: Bool
+      - name: isViewer
+        externalId: isViewer
+        type: Bool
+      - name: location
+        externalId: location
+        type: String
+      - name: name
+        externalId: name
+        type: String
+      - name: pinnedItemsRemaining
+        externalId: pinnedItemsRemaining
+        type: Int64
+      - name: projectsResourcePath
+        externalId: projectsResourcePath
+        type: String
+      - name: resourcePath
+        externalId: resourcePath
+        type: String
+      - name: twitterUsername
+        externalId: twitterUsername
+        type: String
+      - name: updatedAt
+        externalId: updatedAt
+        type: DateTime
+      - name: viewerCanChangePinnedItems
+        externalId: viewerCanChangePinnedItems
+        type: Bool
+      - name: viewerCanCreateProjects
+        externalId: viewerCanCreateProjects
+        type: Bool
+      - name: viewerCanFollow
+        externalId: viewerCanFollow
+        type: Bool
+      - name: viewerIsFollowing
+        externalId: viewerIsFollowing
+        type: Bool
+relationships:
```

```suggestion
        type: Bool
    
relationships:
```
nit: Add new line between entity and relationship

---

I'll address this one in the follow-up PR